dyson_compress_1403.xml

```diff
@@ -326,7 +326,7 @@
     <port label="release_time" dir="input" type="control" hint="default_low">
       <name>Release time (s)</name>
       <p>Controls the time taken for the compressor to relax its gain control over the input signal.</p>
-      <range min="0" max="1"/>
+      <range min="0.0000001" max="1"/>
```

Comment:
I chose this number to be small enough for the value on the knobs in lmms to round off to 0. We do some odd math for our knob values so this need to be tested. The value is only used in one place though so it may be cleaner to change:
`float rgainfilter = 1.0f / (release_time * sample_rate);` to something like:
`float rgainfilter = 1.0f / f_max(release_time * sample_rate, 1.0f);`